src/Teams/Services/ManageTeamsMembersService.cs

```diff
@@ -0,0 +1,41 @@
+using Teams.Models;
+using Teams.Data;
+using System.Linq;
+using Teams.Services;
+using Teams.Security;
+using Microsoft.AspNetCore.Authorization;
+using System.Threading.Tasks;
+
+namespace Teams.Services
+{
+    public class ManageTeamsMembersService : IManageTeamsMembersService
+    {
+        private readonly ICurrentUser _currentUser;
+        private readonly IRepository<Team, int> _teamRepository;
+        private readonly IRepository<TeamMember, int> _memberRepository;
+
+        public ManageTeamsMembersService(IRepository<Team, int> teamRepository, IRepository<TeamMember, int> memberRepository, ICurrentUser currentUser)
+        {
+            _currentUser = currentUser;
+            _teamRepository = teamRepository;
+            _memberRepository = memberRepository;
+        }
+
+        [Authorize]
```

Comment:
Why we use Authorize attribute in bussines layer? It should be located only on controller level or action level